db/data_migration/20150424094040_republish_coming_soon.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(Edition.scheduled).perform
```

Comment:
From reading `Whitehall::PublishingApi.schedule_async`, it seems unnecessary to reschedule the editions, we just want to resend the `coming_soon` items. So this migration should be:
```ruby
Edition.scheduled.each do |edition|
  unless edition.document.published?
    edition.translated_locales.each do |locale|
      PublishingApiComingSoonWorker.perform_async(edition.id, locale)
    end
  end
end
```